cardano-cli/test/Test/CLI/TextEnvelope/Golden/Genesis/Create.hs

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE OverloadedStrings    #-}
+
+module Test.CLI.TextEnvelope.Golden.Genesis.Create
+  ( golden_genesisCreate
+  ) where
+
+import Cardano.Prelude hiding (to)
+
+import Data.Maybe
+  ( fromJust
+  )
+import Control.Lens
+  ( (^?)
+  , (^..)
+  , each
+  )
```

Comment:
We have tended to avoid using `lens` across the whole Cardano project. JSON might be one of the few exceptions to this.

Also you should look at some of the other files in the project for formattiing/layout.

Reply:
Is there a good example file I should look at?

Reply:
@Jimbo4350 ? Maybe you could point to the file with the formatting that is closest to what you would like.

Reply:
`STYLE.md` is a good start but honestly I think we need to come to consensus about using a code formatter. None are ideal but I would take consistency over perfection in this case.

Reply:
I endorse the idea of using a code formatter.